src/p2p/net_node.inl

```diff
@@ -1885,6 +1886,7 @@ namespace nodetool
   bool node_server<t_payload_net_handler>::gray_peerlist_housekeeping()
   {
     if (!m_exclusive_peers.empty()) return true;
+    if (m_offline) return true;
```

Comment:
Starting to nitpick again.. For consistency, it would be better to move this up a line.